_posts/2020-10-08-how-to-effectively-manage-school-a-business.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: 21. How to effectively manage school & a business
+external-url: https://www.buzzsprout.com/1341670/episodes/5800249
+---
+
+## {{page.title}}
+
+OCTOBER 8, 2020
```

Comment:
@tslevy , the previous episodes have leading zeros, so let's stick with that to be consistent for now.